fly/commands/hijack.go

```diff
@@ -63,10 +64,12 @@ func (command *HijackCommand) Execute([]string) error {
 	if err != nil {
 		return err
 	}
+
 	var chosenContainer atc.Container
 
 	if command.Handle != "" {
-		chosenContainer, err = target.Team().GetContainer(command.Handle)
+		team := command.TeamFlag.GetTeamTarget(target)
```

Comment:
tiny idea: since `TeamFlag` is embedded, this could just be `command.GetTeamTarget(target)`